README.md

```diff
@@ -91,6 +91,27 @@ bazel
 
 Bazelisk itself should not require further updates, as its only purpose is to keep Bazel updated.
 
+## zlib on Linux
+Linux distributions, in general, use the original zlib. To confirm this please run the objdump command against the zlib shared object file. For example, on Amazon Linux 2, run:
+```
+yum install zlib-devel
+objdump -D /usr/lib64/libz.so | grep inflate_fast_chunk
+```
+If there are no results, then the original zlib is being used.
+
+zlib-cloudflare has been updated to provide better and faster compression on Arm and x86. To use zlib-cloudflare:
+```
+git clone https://github.com/cloudflare/zlib.git
+cd zlib
+./configure
+make
+make install
```

Comment:
here you're effectively saying override the system installed zlib package artifacts with this one manually. That isn't best practices.

Reply:
Updated to ./configure --prefix=$HOME. This will cause make install to install the file to $HOME.